.setup/install_submitty/install_site.sh

```diff
@@ -38,6 +38,17 @@ set_permissions () {
     esac
 }
 
+set_mjs_permission () {
+    for file in $1/*; do
+        if [ -d "$file" ]; then
+            chmod 551 $file
```

Comment:
Is there a specific reason for using `551`, I think only the php site user needs execute permissions on JS so there wouldn't be a need for the user or group to execute it?

Reply:
The chmod here is only setting the permissions to `551` if it is a directory (if it is a file it calls `set_permissions` like before); I'm not sure what permissions child directories need though and I just assumed it based on the permissions set on the `mjs` directory previously.